stdlib/test/builtin/test_file.mojo

```diff
@@ -68,16 +68,19 @@ def test_file_read_bytes_multi():
 
     # CHECK: Lorem ipsum
     var bytes1 = f.read_bytes(12)
+    assert_equal(bytes1, "Lorem ipsum")
     print(String(bytes1))
 
     # CHECK: dolor
```

Comment:
**Suggestion** We can drop this `CHECK` and the `print` below your new assert.  Ditto below for the `print(len(s))` and its `CHECK` line.

Reply:
The unnecessary prints and comments are now dropped.